### DISCUSSION TITLE
How to use peer in Twind?

### DISCUSSION THREAD

**Discussion by logneh4** [freshframework/fresh#1465] [Q&A] [answered]
Flowbite and Tailwind share `peer-` class to control next element if input is checked or invalid:
![image](https://github.com/denoland/fresh/assets/116230374/bb6d1833-d4d9-4fda-96e2-4421e2821593)

but this doesnt work in twind. Can I do something?

**Comment by marvinhagemeister** [comment, marked as answer]
Twind doesn't support the `peer` classes. Starting with the upcoming Fresh 1.6 release we'll ship with a proper Tailwind CSS plugin which uses the same npm package as you'd use with node.
